plugins/Morpheus/stylesheets/general/_admin.less

```diff
@@ -1,4 +1,8 @@
 
+.admin {
+  width: 100%;
```

Comment:
Using global classes is always a bit "risky". This specific class would e.g. also apply here:
https://github.com/matomo-org/matomo/blob/9a24d934841ebc39b9443dd2d47f5acc9f9d5e61/plugins/Feedback/templates/index.twig#L10
It doesn't hurt in this case, as it doesn't have an effect, but might also be possible that a plugin might use that class for anything.
As it should only effect the admin content page, it might be better placed here:
https://github.com/matomo-org/matomo/blob/d13d4c4db7be2625e2aedf415f58de9bcb70a81b/plugins/CoreHome/stylesheets/coreHome.less#L23-L30